packages/issuer-sdk-js/src/crypto.ts

```diff
@@ -0,0 +1,18 @@
+import * as base64 from "@stablelib/base64";
+import * as utf8Codec from "@stablelib/utf8";
+import nacl from "tweetnacl";
+
+export async function encrypt(message: string, encryptionPublicKey: string, secretKey: string) {
```

Comment:
Consider pushing the decoding of values to the edges of the application.

```suggestion
export async function encrypt(message: Uint8Array, encryptionPublicKey: Uint8Array, secretKey: Uint8Array) {
```